docs/src/prototypes/chatPane/services/messageFactoryMock.tsx

```diff
@@ -44,6 +46,53 @@ function generateChatMsgProps(msg: MessageData, fromUser: UserData): ChatMessage
   return msgProps
 }
 
+function getMessageContent(content: string) {
+  const contextMenu = (
+    <Menu
+      items={[
+        { key: 'download', content: 'Download', icon: 'download' },
+        { key: 'linkify', content: 'Get link', icon: 'linkify' },
+        { key: 'tab', content: 'Make this a tab', icon: 'folder open' },
+      ]}
+      vertical
+      pills
+    />
+  )
+
+  const actionPopup = (
+    <Popup
+      accessibility={popupFocusTrapBehavior}
+      trigger={
+        <Button aria-label="More attachment options" iconOnly circular icon="ellipsis horizontal" />
+      }
+      content={{ content: contextMenu }}
+    />
+  )
+
+  return (
+    <>
+      <span>
+        {content} <a href="/"> Some link </a>
+      </span>
+      <div style={{ marginTop: '20px' }}>
+        {_.map(['MeetingNotes.pptx', 'Document.docx'], (fileName, index) => {
+          return (
+            <Attachment
```

Comment:
We don't need there `return`:

```diff
- {_.map(['MeetingNotes.pptx', 'Document.docx'], (fileName, index) => {
-          return (
-            <Attachment
+{_.map(['MeetingNotes.pptx', 'Document.docx'], (fileName, index) => (
+  <Attachment
```

This will allow to reduce the nesting level 👍